File: ALittle/Module/ALittleIDE/Other/GameLibrary/CEngine/JSScript/CEngine/UISystem/Base/DisplayView.js
{
if (typeof ALittle === "undefined") ALittle = {};


if (ALittle.DisplayLayout === undefined) throw new Error(" extends class:ALittle.DisplayLayout is undefined");
ALittle.DisplayView = JavaScript.Class(ALittle.DisplayLayout, {
	Ctor : function(ctrl_sys) {
		this._show = ALittle.NewObject(lua.__CPPAPIDisplayView);
		this._pickup_rect = true;
		this._pickup_child = true;
	},
}, "ALittle.DisplayView");

}

File: ALittle/Module/ALittleIDE/Other/GameLibrary/CEngine/Script/CEngine/UISystem/Base/DisplayView.lua

module("ALittle", package.seeall)

local ___rawset = rawset
local ___pairs = pairs
local ___ipairs = ipairs


assert(ALittle.DisplayLayout, " extends class:ALittle.DisplayLayout is nil")
DisplayView = Lua.Class(ALittle.DisplayLayout, "ALittle.DisplayView")

function DisplayView:Ctor(ctrl_sys)
	___rawset(self, "_show", __CPPAPIDisplayView())
	___rawset(self, "_pickup_rect", true)
	___rawset(self, "_pickup_child", true)
end

